src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1412,6 +1412,13 @@ def load_arguments(self, _):
         c.argument('ppg_type', options_list=['--type', '-t'], help="The type of the proximity placement group. Allowed values: Standard.")
         c.argument('tags', tags_type)
 
+    with self.argument_context('ppg create', min_api='2021-11-01') as c:
+        c.argument('zone', nargs='*', help="Specifies the availability zone where virtual machine, virtual machine scale set or availability set associated with the proximity placement group can be created. Accepted values 1,2,3.")
```

Comment:
Please use the first person voice in the help message
```suggestion
        c.argument('zone', nargs='*', help="Specify the availability zone where virtual machine, virtual machine scale set or availability set associated with the proximity placement group can be created. Accepted values 1,2,3.")
```